src/nft/pages/asset/Asset.tsx

```diff
@@ -37,11 +39,13 @@ const AssetPriceDetailsContainer = styled.div`
 const AssetPage = () => {
   const { tokenId = '', contractAddress = '' } = useParams()
   const { data, loading } = useNftAssetDetails(contractAddress, tokenId)
+  const detailsV2Enabled = useDetailsV2Enabled()
 
   const [asset, collection] = data
 
-  if (loading) return <AssetDetailsLoading />
+  if (detailsV2Enabled) return <NftDetails asset={asset} collection={collection} />
```

Comment:
Eventually we'll want this page under the same Trace analytics component as the current `AssetContainer`, could you use the `detailsV2Enabled` conditional to switch between that component and the new `NftDetails` component?